src/pyodide/internal/python.js

```diff
@@ -139,7 +146,12 @@ function getEmscriptenSettings(lockfile, indexURL) {
     // jsglobals is used for the js module.
     jsglobals: globalThis,
     // environment variables go here
-    env: { HOME: "/session" },
+    env: {
+      HOME: "/session",
+      // We don't have access to cryptographic rng at startup so we cannot support hash
+      // randomization. Setting `PYTHONHASHSEED` disables it.
+      PYTHONHASHSEED: "111",
```

Comment:
Likewise, not necessary if using a memory snapshot.

Reply:
Does this mean we can avoid the error handling in our custom `getRandomValues`?